CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 4.1.15 (Not released)
+
+**Bug fixes:**
+
+- Removed PBS and SGE platforms as they are not working in AS4 #2349
```

Comment:
@dbeltrankyl what about using just the hash and the issue number? It should work on GH, with the advantage that the link is relative to current project (so if the org/project are renamed or moved within GH that should still work), and they are a bit easier/faster to write :grimacing: